source/game/field/KColData.hh

```diff
@@ -86,14 +88,19 @@ private:
     f32 m_radius;
     KCLTypeMask m_typeMask;
     const u16 *m_prismIter;
-    u32 m_prismCount;
-    const KCollisionPrism *m_prisms;
+    const KCollisionPrism *m_prismPtr;
     EGG::BoundBox3f m_bbox;
     std::array<u16, 256> m_prismCache;
     u16 *m_prismCacheTop;
     u16 *m_cachedPrismArray;
     EGG::Vector3f m_cachedPos;
     f32 m_cachedRadius;
+
+    /// @brief This is a performance enhancement for Kinoko in order to avoid repeated calls to parse(). Since we don't have the same memory limitations as the Wii,
+    /// let's increase prism lookup performance at the cost of increased memory usage.
```

Comment:
```suggestion
    /// @brief Optimizes for time by avoiding unnecessary byteswapping.
    /// The Wii doesn't have this problem because big endian is always assumed.
```